config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   resources :sites do
-    resources :reviews
+    resources :reviews, only: [:index, :new, :create, :edit, :update, :destroy]
+  end
+
+  resources :reviews, only: [:show] do
+    resources :votes, only: [:create, :update, :destroy]
```

Comment:
consider creating different controllers for upvotes and downvotes and posting to urls like:

```no-highlight
/reviews/23132/upvotes

/reviews/213214/downvotes
```